GiEnJul/ClientApp/src/components/admin/suggestedConnections/RecipientTable.tsx

```diff
@@ -0,0 +1,175 @@
+        <TableCell style={{ paddingBottom: 0, paddingTop: 0 }} colSpan={6}>
+          <Collapse in={open} timeout="auto">
+            <Box margin={1}>
+              <Typography variant="h6" gutterBottom component="div">
+                Matønsker
+              </Typography>
+              <Table size="small" aria-label="purchases">
+                <TableHead></TableHead>
+                <TableBody>
+                  <TableRow>
+                    <TableCell>{capitalize(recipient.dinner)}</TableCell>
+                    <TableCell>{capitalize(recipient.dessert)}</TableCell>
+                    <TableCell>{recipient.note}</TableCell>
+                  </TableRow>
+                </TableBody>
+              </Table>
+              <Typography variant="h6" gutterBottom component="div">
+                Familiemedlemmer
+              </Typography>
+              <Table size="small" aria-label="purchases">
+                <TableHead>
+                  <TableRow>
+                    <TableCell>Kjønn</TableCell>
+                    <TableCell>Alder</TableCell>
+                    <TableCell>Ønske</TableCell>
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {recipient.familyMembers.map((familyMember) => (
+                    <TableRow key={familyMember.rowKey}>
+                      <TableCell>{getGender(familyMember.gender)}</TableCell>
+                      <TableCell>{familyMember.age}</TableCell>
+                      <TableCell component="th" scope="row">
+                        {familyMember.wish == null
+                          ? "Giver kjøper alderstilpasset gave"
+                          : familyMember.wish}
+                      </TableCell>
+                    </TableRow>
+                  ))}
+                </TableBody>
+              </Table>
+              <Typography variant="h6" gutterBottom component="div">
+                Kontakt
+              </Typography>
+              <Table size="small" aria-label="purchases">
+                <TableHead></TableHead>
+                <TableBody>
+                  <TableRow>
+                    <TableCell>{recipient.contactFullName}</TableCell>
+                    <TableCell>{recipient.contactEmail}</TableCell>
+                    <TableCell>{recipient.contactPhoneNumber}</TableCell>
+                  </TableRow>
+                </TableBody>
+              </Table>
+            </Box>
+          </Collapse>
+        </TableCell>
+      </TableRow>
+    </>
+  );
+}
+
+type TableProps = {
+  selectRecipient: (newSelected: [string, string]) => void;
+};
+
+export default function RecipientSuggestions(props: TableProps) {
+  const [recipients, setRecipients] = useState<RecipientType[]>([]);
+  const [selected, setSelected] = useState<string>("");
+
+  useEffect(() => {
+    api
+      .get("admin/Suggestions/Recipient", {
+        params: { location: "Stavanger" /*TODO: Fetch by user.location*/ },
```

Comment:
This can be fetched now by using the useUser hook that's on master

Reply:
fixed :)